### ISSUE TITLE
Is there a guide to install on VPS/server?

### ISSUE THREAD

**Issue by ferritopia** [snibox/snibox#77]
I am having problem during installation to my VPS, I cannot access is from neither IP address nor domain name. I followed documentation for Ubuntu, but I guess that's for personal computer and not for server. I works on my computer tho, but not on my VPS.

P.S I have zero knowledge of coding. 
Help?

**Comment by piv-pav** [comment]
I don't think this is the place to discuss the way to install Ruby on Rails application, as your issue is general RoR question, not a project specific, however the easiest way on my taste would be using docker images [snibox/snibox](https://hub.docker.com/r/snibox/snibox) or [snowmean/snibox-sqlite](https://hub.docker.com/repository/docker/snowmean/snibox-sqlite) in case if you don't want to use PostgreSQL\MariaDB.

```
mkdir ~/snibox-db
docker run -d -p 80:3000 -v ~/snibox-db:/app/db/database snowmean/snibox-sqlite
```
